whois-update/src/main/java/net/ripe/db/whois/update/dns/zonemaster/domain/StartDomainTestRequest.java

```diff
@@ -52,6 +52,9 @@ public void setClientVersion(final String version) {
         this.params.setClientVersion(version);
     }
 
+    public Params getParams(){
+        return this.params;
```

Comment:
Why you needed a getter for param? I don't see it being called?

Reply:
It is calling by the unit test